src/JSystem/J3DGraphBase/J3DSys.cpp

```diff
@@ -123,7 +123,7 @@ void J3DFifoLoadNrmMtxImm3x3(Mtx33 mtx, unsigned long idx) {
     GFX_FIFO(u32) = mtx_u32[8];
 }
 
-u8 NullTexData[16] = {
+u8 NullTexData[16] __attribute__((aligned(32))) = {
```

Comment:
Not sure what type this is supposed to be, but the map & symbols.txt have it listed as `align:32`, which is odd. Required to get the section alignment correct.